source/guides/stacks/dlrs.rst

```diff
@@ -540,15 +540,14 @@ To run the notebook, you will need to run the Deep Learning Reference Stack, mou
    .. code-block:: bash
 
       cd workspace
-      git clone https://github.com/huggingface/transformers.git
+      git clone https://gist.github.com/16d38f2c9c688963c166c000330a3c11.git
 
 
 
-#. Navigate to the Transformers notebook directory, and start a Jupyter Notebook that is linked to the exterior port.  Be sure to copy the token from the output of starting  Jupyter Notebook.
+#. Start a Jupyter Notebook that is linked to the exterior port.  Be sure to copy the token from the output of starting  Jupyter Notebook.
 
    .. code-block:: bash
-
-      cd transformers/notebooks
+      pip install jupyter --upgrade
```

Comment:
Correction: Add an empty line before `pip install ...` 
Then make sure command for `jupyter notebook` appears on the next line immediately following it. 

This is the reason for failure to pass CI/CD.

Reply:
Thanks @mvincerx , on it!